recipe/conda_build_config.yaml

```
@@ -1,4 +1,19 @@
-MACOSX_SDK_VERSION:            # [osx and x86_64]
-  - "10.13"                    # [osx and x86_64]
-MACOSX_DEPLOYMENT_TARGET:      # [osx and x86_64]
-  - "10.13"                    # [osx and x86_64]
+MACOSX_SDK_VERSION:            # [osx and x86_64 and qt_version!=6]
+  - "10.13"                    # [osx and x86_64 and qt_version!=6]
+MACOSX_DEPLOYMENT_TARGET:      # [osx and x86_64 and qt_version!=6]
+  - "10.13"                    # [osx and x86_64 and qt_version==none]
+  - "10.13"                    # [osx and x86_64 and qt_version==5]
```

Do you foresee these two to diverge at some point? Otherwise, why not keep it in line with the other selectors?
```suggestion
  - "10.13"                    # [osx and x86_64 and qt_version!=6]
```

---

thanks